src/gettsim/plot/dag/__init__.py

```diff
@@ -2,10 +2,46 @@
 
 from typing import TYPE_CHECKING
 
-from ttsim import plot as ttsim_plot
+import ttsim
 
 from gettsim import germany
 
+GETTSIM_COLORMAP = {
+    # Top-level, background variables - blue.
+    ("top-level",): "lightskyblue",
+    ("einnahmen",): "mediumblue",
+    ("familie",): "skyblue",
+    ("wohnen",): "mediumturquoise",
+    ("hh_characteristics",): "skyblue",
+    ("ids",): "deepskyblue",
+    ("unterhalt",): "teal",
+    # Taxes - red. Exception: Einkünfte are mix of Einnahmen/Tax rules - purple.
+    ("einkommensteuer",): "crimson",
+    ("einkommensteuer", "einkünfte"): "purple",
+    ("lohnsteuer",): "red",
+    ("solidaritätszuschlag",): "darkred",
+    # Social insurance - differentiate between programs and between pension
+    # contributions and pension benefits.
+    ("sozialversicherung",): "gold",
+    ("sozialversicherung", "arbeitslosen"): "yellowgreen",
+    ("sozialversicherung", "kranken"): "yellow",
+    ("sozialversicherung", "pflege"): "khaki",
+    ("sozialversicherung", "rente"): "goldenrod",
+    ("sozialversicherung", "rente", "beitrag"): "darkgoldenrod",
+    # Transfers - green
+    ("kindergeld",): "olive",
+    ("kinderbonus",): "darkolivegreen",
+    ("kinderzuschlag",): "olivedrab",
+    ("elterngeld",): "darkgreen",
+    ("erziehungsgeld",): "darkgreen",
+    ("unterhaltsvorschuss",): "seagreen",
+    ("wohngeld",): "turquoise",
```

Comment:
```suggestion
    ("wohngeld",): "darkseagreen",
```

`turquoise` is very blue...

Reply:
Also agreed, and "Wohngeld" and "Kinderzuschlag" still look different enough with this change.